index.html

```diff
@@ -107,6 +179,25 @@ <h2>
             </button>
         </section>
         <footer>
+            <div class="partners">
+                <h2>
+                    Partner
+                </h2>
+                <img src="assets/images/guide-bar.svg" alt="guide bar image" class="guide-bar-logo">
+                <div class="partners-logo">                    
+                    <img src="assets/images/mozilla.png" alt="mozilla logo" id="mozilla">
+                    <img src="assets/images/google.png" alt="google logo" id="google">
+                    <img src="assets/images/naver.png" alt="naver logo" id="naver">
+                    <img src="assets/images/kakao.png" alt="kakao logo" id="kakao">
+                    <img src="assets/images/airbnb.png" alt="airbnb logo" id="airbnb">
+                </div>
+            </div>
```

Comment:
- Kindly, those partner logos are lists so make it inside `ul` and each items inside `li` to make more semantically correct 👍👍